source/manual/encrypted-hiera-data.html.md

```diff
@@ -418,3 +419,17 @@ for the development VM:
 
 Alternatively, you can add the `$LOAD_PATH` to `/usr/bin/puppet` as shown
 in [this commit](https://github.com/alphagov/govuk-puppet/commit/b7743452875b1dd83fda982e28ae8e776bc3a8b8).
+
+### Zsh: no matches found
+
+If you encounter an error similar to
+
+```
+zsh: no matches found: eyaml:edit[integration]
```

Comment:
I wonder if it's easier to just make all the examples in this page have quotes around the argument because I believe that should work the same under Zsh and Bash: `bundle exec rake 'eyaml:edit[integration]'`.

Reply:
I still think there's value in calling out the actual error so when searching for the error string, we get an answer.
I don't know about quotes around the arguments - I agree it's valid, but perhaps not typical/expected when working with rake?

Reply:
Good point, the error is still useful. Perhaps, it's just me that uses the quotes!